packages/cli/lib/sandboxes.js

```diff
@@ -51,20 +58,38 @@ const sandboxApiTypeMap = {
   developer: 2,
 };
 
-const getSandboxTypeAsString = type =>
-  type === 'DEVELOPER' ? 'development' : 'standard';
+const getSandboxTypeAsString = accountType => {
+  if (
+    accountType === HUBSPOT_ACCOUNT_TYPES.DEVELOPMENT_SANDBOX ||
+    accountType === 'DEVELOPER'
+  ) {
+    return 'development';
+  }
+  return 'standard';
+};
 
-const isSandbox = config =>
+const getSandboxName = config =>
+  `[${getSandboxTypeAsString(
+    config.accountType || config.sandboxAccountType
+  )} sandbox] `;
+
+const isSandbox_OLD = config =>
   config.sandboxAccountType && config.sandboxAccountType !== null;
 
-function getAccountName(config, bold = true) {
-  const sandboxName = `[${getSandboxTypeAsString(
-    config.sandboxAccountType
-  )} sandbox] `;
+const isSandbox = config =>
+  config.accountType
+    ? config.accountType === HUBSPOT_ACCOUNT_TYPES.STANDARD_SANDBOX ||
+      config.accountType === HUBSPOT_ACCOUNT_TYPES.DEVELOPMENT_SANDBOX
+    : isSandbox_OLD(config);
 
-  const message = `${config.name} ${isSandbox(config) ? sandboxName : ''}(${
-    config.portalId
-  })`;
+function getAccountName(config, bold = true) {
+  const message = `${config.name} ${
+    isSandbox(config)
+      ? getSandboxName(config)
+      : isDeveloperTestAccount(config)
```

Comment:
Added developer test account support here so we get the correct hub tag in other command usages

Reply:
We have a [ui util for logging the user-friendly account name](https://github.com/HubSpot/hubspot-cli/blob/master/packages/cli/lib/ui.js#L73). It's pretty similar to what this is doing (but currently isn't very smart about handling different account types). Seems like maybe we could consolidate the functionality so we just have one util that we can use everywhere.

Reply:
ah nice, yeah I can move the logic over to that util so everyone gets it